tests/e2e/etcd_mix_versions_test.go

```diff
@@ -17,6 +17,7 @@ package e2e
 import (
 	"context"
 	"fmt"
+	"github.com/stretchr/testify/require"
```

Comment:
please reorder this import item. Put it at the place together with `github.com/stretchr/testify/assert`

Reply:
done